scripts/influxdb_import.py

```diff
@@ -14,7 +14,12 @@
 
 influxdb = None
 try:
-    influxdb = InfluxDBClient(host='biomaj-influxdb', database='biomaj')
+    host = BiomajConfig.global_config.get('GENERAL', 'influxdb.host')
+    user = BiomajConfig.global_config.get('GENERAL', 'influxdb.user')
+    password = BiomajConfig.global_config.get('GENERAL', 'influxdb.password')
+    port = BiomajConfig.global_config.get('GENERAL', 'influxdb.port')
+    database = BiomajConfig.global_config.get('GENERAL', 'influxdb.db')
+    influxdb = InfluxDBClient(host=host, database=database, port=port, username=user, password=password)
```

Comment:
looking at influxdb code, clilent maps automatically to int, so no issue. Gonna merge request